### PR TITLE
Toast component

### PR DESCRIPTION
**Pull request by minhtuev**
## What changes are proposed in this pull request?

Added a reusable toast component that acts like a notification for users that can prompt users for actions and disappear within a certain minute

Example:
image

## How is this patch tested? If it is not, please explain why.

* Local testing (✅ )
* Unit/e2e tests

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **New Features**
	- Introduced a customizable `Toast` notification component for displaying messages.
	- The `Toast` component supports an action button, customizable message, and auto-hide duration.

- **Enhancements**
	- Added centralized export for the `Toast` component, making it easily accessible from other parts of the application.

### CODE REVIEW

File: app/packages/components/src/components/index.ts [hunk 1]
```diff
@@ -37,5 +37,6 @@ export { default as TabOption } from "./TabOption";
 export { default as TextField } from "./TextField";
 export { default as ThemeProvider, useFont, useTheme } from "./ThemeProvider";
 export { default as Tooltip } from "./Tooltip";
+export { Toast } from "./Toast";
```
Comment (Br2850) [inline app/packages/components/src/components/index.ts:40]:
```suggestion
export { default as Toast } from "./Toast";
```
Can we follow the existing pattern here, to keep this file consistent?

File: app/packages/components/src/components/Toast/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export {default as Toast} from "./Toast";
```
Comment (Br2850) [inline app/packages/components/src/components/Toast/index.ts:1]:
```suggestion
export { default } from "./Toast";
```
See other comment about consistency

**Comment by CamronStaley** [comment]
Do we have storybooks for UI components like this? Would be cool to be able to test it in isolation and if all our components have them would be great for people that don't know of all our reusable components (like myself).

**Review by Br2850 — APPROVED** [review]
LGTM 👍🏽 pretty sure you intended this to be just an `app` component so if that is the case it looks good to me. 2 minor consistency changes suggested, will help with code posterity

**Review by CamronStaley — APPROVED** [review]
Talked offline about storybooks - LGTM

**Review by ritch — APPROVED** [review]

**Pull request merged**